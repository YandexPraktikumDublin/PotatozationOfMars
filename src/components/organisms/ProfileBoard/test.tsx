import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ProfileBoard } from '.'

describe('<ProfileBoard />', () => {
  it('should renders correct <ProfileBoard />', () => {
    const wrapper = shallow(<ProfileBoard />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
