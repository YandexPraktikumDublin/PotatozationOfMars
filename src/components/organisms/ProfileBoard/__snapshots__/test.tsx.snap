// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileBoard /> should renders correct <ProfileBoard /> 1`] = `
<div
  className="text-primary dark:text-white"
>
  <List
    className="mb-12"
  >
    <NameValueListItem
      name="Email"
      value="ivan@yandex.ru"
    />
    <NameValueListItem
      name="Login"
      value="IvanIvanov"
    />
    <NameValueListItem
      name="First name"
      value="Ivan"
    />
    <NameValueListItem
      name="Last name"
      value="Ivanov"
    />
    <NameValueListItem
      name="Phone number"
      value="+790000000000"
    />
    <NameValueListItem
      name="Password"
      value="••••••"
    />
  </List>
  <List>
    <ActionsListItem
      name="Log out"
      onClick={[Function]}
    />
  </List>
</div>
`;
